catfs/fs.go

```diff
@@ -32,9 +32,14 @@ import (
 )
 
 const (
-	abiVersion = 1
+	abiVersion                 = 1
+	defaultEncryptionKeyLength = 32
```

Comment:
Just a musing: It's a pure coincidence that key length is always 32 bytes, since both algorithms we support are 256bit. If we ever change that we have to make sure that we support different key sizes and make the logic depend on the chosen algorithm.

Nothing to do with this PR, just mentioning because I was reminded of that.

Reply:
This one is a bit harder to correct. The definition of this constant might
belong to encryption stream code. Since you don't object, I leave it as is for now.